snapshots/src/main/java/io/axoniq/dev/samples/Application.java

```diff
@@ -0,0 +1,32 @@
+package io.axoniq.dev.samples;
+
+import org.axonframework.eventsourcing.EventCountSnapshotTriggerDefinition;
+import org.axonframework.eventsourcing.SnapshotTriggerDefinition;
+import org.axonframework.eventsourcing.Snapshotter;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+/**
+ * @author Sara Pellegrini
+ * @author Lucas Campos
+ */
+@SpringBootApplication
+public class Application {
+
+    public static void main(String[] args) {
+        SpringApplication.run(Application.class, args);
+    }
+
+    /**
+     * This {@link SnapshotTriggerDefinition} is responsible to trigger the Snapshot. In this implementation, it creates
+     * a Snapshot every 5 events.
```

Comment:
I think we should explain a little deeper, it is each 5 events, but the snapshot event is included in the count.

Reply:
Added some examples.